media-video/obs-studio/obs-studio-18.0.2.ebuild

```diff
@@ -51,30 +51,32 @@ DEPEND="
 "
 RDEPEND="${DEPEND}"
 
-src_prepare() {
-	CMAKE_REMOVE_MODULES_LIST=(FindFreetype)
-
-	cmake-utils_src_prepare
-}
+CMAKE_REMOVE_MODULES_LIST=( FindFreetype )
 
 src_configure() {
-	local libdir="$(get_libdir)"
+	local libdir=$(get_libdir)
 	local mycmakeargs=(
-		-DDISABLE_ALSA="$(usex !alsa)"
-		-DDISABLE_FREETYPE="$(usex !truetype)"
-		-DDISABLE_JACK="$(usex !jack)"
-		-DDISABLE_LIBFDK="$(usex !fdk)"
-		-DDISABLE_PULSEAUDIO="$(usex !pulseaudio)"
-		-DDISABLE_V4L2="$(usex !v4l)"
-		-DLIBOBS_PREFER_IMAGEMAGICK="$(usex imagemagick)"
-		-DOBS_MULTIARCH_SUFFIX="${libdir#lib}"
+		-DDISABLE_ALSA=$(usex !alsa)
+		-DDISABLE_FREETYPE=$(usex !truetype)
+		-DDISABLE_JACK=$(usex !jack)
+		-DDISABLE_LIBFDK=$(usex !fdk)
+		-DDISABLE_PULSEAUDIO=$(usex !pulseaudio)
+		-DDISABLE_V4L2=$(usex !v4l)
+		-DLIBOBS_PREFER_IMAGEMAGICK=$(usex imagemagick)
+		-DOBS_MULTIARCH_SUFFIX=${libdir#lib}
+		-DOBS_VERSION_OVERRIDE=${PV}
 		-DUNIX_STRUCTURE=1
 	)
-
 	cmake-utils_src_configure
 }
 
+pkg_preinst() {
+	gnome2_icon_savelist
```

Comment:
This is no longer necessary, due to the most recent gnome2-utils eclass changes.